src/components/operations/operation-details/ko/runtime/operation-details.ts

```diff
@@ -244,6 +256,19 @@ export class OperationDetails {
         this.working(false);
     }
 
+    public async loadExamples(operation: Operation): Promise<void> {
+        const representations = operation.request.meaningfulRepresentations().map(a => {return {...a}});
+        let requestExamples = {};
+        if (representations && representations.length) {
+            for(var i = 0; i < representations.length; i++) {
+                let value = representations[i].examples?.[0];
+                if (!value) return;
```

Comment:
Why do you finish the function here? should you not just skip the iteration?

Reply:
Ideally it should not be the case, but let me rewrite it, thanks